plugins/ProfessionalServices/tests/Unit/PromoWidgetApplicableTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\ProfessionalServices\tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use Piwik\Plugin\Manager;
+use Piwik\Plugins\ProfessionalServices\PromoWidgetApplicable;
+use Piwik\ProfessionalServices\Advertising;
+
+class PromoWidgetApplicableTest extends TestCase
+{
+    /**
+     * @dataProvider checkDataProvider
+     */
+    public function test_check_shouldReturnBool(bool $adsForProfessionalServicesEnabled, bool $pluginActivated, bool $expected): void
```

Comment:
A more descriptive test name could be helpful when debugging future test fails, it might return a bool, but is it the correct bool? :slightly_smiling_face: 
```suggestion
    public function test_check_shouldOnlyReturnTrue_IfAdShouldBeShown(bool $adsForProfessionalServicesEnabled, bool $pluginActivated, bool $expected): void
```